casepro/backend/junebug.py

```diff
@@ -40,6 +46,74 @@ def get_addresses(self, uuid):
         return (
             a['address'] for a in addresses if a.get('address') is not None)
 
+    def get_identities(self, **kwargs):
+        '''Get the list of identities filtered by the given kwargs.'''
+        url = '%s/api/v1/identities/search/?' % (self.base_url)
```

Comment:
Just checking whether you intended to write `(self.base_url,)` here instead of `(self.base_url)`?